binaries/cli/Cargo.toml

```diff
@@ -57,6 +57,8 @@ crossterm = "0.29.0"
 ratatui = "0.29.0"
 itertools = "0.14"
 sysinfo = "0.36.1"
+indicatif = "0.17"
+atty = "0.2"
```

Comment:
The `atty` crate is [deprecated](https://github.com/softprops/atty?tab=readme-ov-file#atty). Please use https://doc.rust-lang.org/std/io/trait.IsTerminal.html instead.